server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -1034,6 +1048,82 @@ func LookupInvocation(env environment.Env, ctx context.Context, iid string) (*in
 	return invocation, nil
 }
 
+func GetCacheScoreCard(ctx context.Context, env environment.Env, req *capb.GetCacheScoreCardRequest) (*capb.GetCacheScoreCardResponse, error) {
+	page := &pgpb.OffsetLimit{Offset: 0, Limit: scoreCardPageSize}
+	if req.PageToken != "" {
+		var err error
+		page, err = paging.DecodeOffsetLimit(req.PageToken)
```

Comment:
nit: I would create a local var and re-assign to page as it makes it more clear what's going on. i.e.
reqPage, err := ...
if err != nil {}
page = reqPage

Reply:
sg, done